medusa/logger/__init__.py

```diff
@@ -261,9 +250,6 @@ def reverse_readlines(filename, skip_empty=True, append_newline=False, block_siz
         block_size (int|None): The block size.
             If int, the file is processed in blocks of the specified size.
             If None, the file is processed at once.
-        reset_offset (bool): Reset the file offset.
-            If True, starts reading from the end of the file.
-            Otherwise, starts reading from where the file current position is.
         encoding (str|None): The encoding for correct block size computation.
```

Comment:
`encoding` can't be `None` anymore, because:
```py
yield line.decode(encoding) + (newline if append_newline else empty)
```
```python-console
>>> 'ש'.encode('utf-8').decode(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: decode() argument 1 must be str, not None
```